churchland_pipeline/acquisition.py

```diff
@@ -71,9 +156,61 @@ class Problem(dj.Part):
         ---
         problem_cause: varchar(255) # (e.g. corrupted data)
         """
+        
+    
+    def quickinsert(monkey, rig, task):
+        """
+        Quickly insert session data
+        """
+        
+        # check keys
+        assert len(lab.Monkey & {'monkey': monkey})==1, 'Unrecognized monkey'
+        assert len(lab.Rig & {'rig': rig})==1,          'Unrecognized rig'
+        assert len(Task & {'task': task})==1,           'Unrecognized task'
+
+        # session dates
+        rawPath = Session.rawpath(rig,task,monkey)
+        dates = sorted(list(os.listdir(rawPath)))
+        dates = [x for x in dates if re.search('\d{4}-\d{2}-\d{2}',x) is not None]
+
+        # import session data (can make task-specific later)
+        for date in dates:
+            
+            sessFiles = os.listdir(rawPath + date + '/')
+            
+            # this will need to be updated for non-blackrock data files
+            if (not any(Session & {'session_date': date})
+                and all([x in os.listdir(rawPath + date + '/') for x in ['speedgoat','blackrock']])):
+
+                    # insert session
+                    Session.insert1((date,monkey,rig,task))
+
+                    # insert users
+                    Session.User.insert1((date,monkey,'njm2149'))
+                    if date >= '2019-11-01':
+                        Session.User.insert1((date,monkey,'emt2177'))
+
+                    # insert notes
+                    notesIdx = [re.search('.*notes\.txt',x) for x in sessFiles]
```

Comment:
since you only need the first one, you can use 
```python
try:
    notesFile = next(x for x in sessFiles if re.search('.*notes\.txt',x))
except StopIteration: